setup.py

```diff
@@ -6,28 +6,46 @@
 
 from setuptools import find_packages, setup
 
-version = '2.2a0'
 
-install_requires = [
+# Package metadata.
+
+name = "django-google-spanner"
+description = "Bridge to enable using Django with Spanner."
+version = "2.2a0"
+# Should be one of:
+# 'Development Status :: 3 - Alpha'
+# 'Development Status :: 4 - Beta'
+# 'Development Status :: 5 - Production/Stable'
+release_status = "Development Status :: 3 - Alpha"
+dependencies = [
     'sqlparse >= 0.3.0',
-    'google-cloud >= 0.34.0',
     'google-cloud-spanner >= 1.8.0',
 ]
+extras = {}
+
+
+# Setup boilerplate below this line.
+
+package_root = os.path.abspath(os.path.dirname(__file__))
```

Comment:
You need to `import os`. The presubmit tests are all failing because `os` is not defined